MusicKeyStrokes/Form1.cs

```diff
@@ -16,5 +17,20 @@ public Form1()
         {
             InitializeComponent();
         }
+
+        private void Form1_Load(object sender, EventArgs e)
+        {
+            //using serializer 
+            JsonSerializer sr = new JsonSerializer();
+            var music = sr.Deserialize<KeyModel>();
+
+            if (music == null)
+                music = new List<KeyModel>();
+
+            var model = new KeyModel() { NameSound = "ban", KeyValue = Keys.Q, Layout = LayoutSound.Valakas1,PathSound = "music/valakas1/ban.mp3" };
+            music.Add(model);
+            sr.Serialize<KeyModel>(music);
```

Comment:
[https://i.kym-cdn.com/photos/images/newsfeed/001/303/345/54e.jpg](url)

Reply:
its just for sample. Will be deleted later